public/scripts/app.js

```diff
@@ -2,6 +2,94 @@ console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
 
-// your code
+// Profile GET Statement
+$.ajax({
+  method: "GET",
+  url:"/api/profile"
+})
+// DB GET Statement
+$.ajax({
+  method: "GET",
+  url: "/api/heroes",
+  success: displayHero
+})
 
+
+// Function to show seeded DB
+function displayHero(heroes){
+  heroes.forEach(function(hero){
+    renderHero(hero);
+  });
+}
+// Post function
+  $("form").on("submit", function(e){
+    e.preventDefault();
+      let userHero = $(this).serialize();
+      console.log(userHero);
+      $.post("/api/heroes", userHero, function(data){
+        renderHero(data);
+      });
+      $(this).trigger("reset");
+});
+// Delete button actions on click event
+$("#hero-list").on("click", ".btn-danger", removeHero());
+// DELETE Ajax Call
+$.ajax({
+  url: "/api/heroes/:Id" + removeHero(),
```

Comment:
This AJAX call is in the middle of your code, not actually attached to the delete button at all, and it uses the literal string ":Id" instead of the actual ID of a hero.